conanfile.py

```diff
@@ -193,6 +193,7 @@ def requirements(self):
         self.options['zlib'].shared = True
         self.requires("libuuid/1.0.3")
         self.options['libuuid'].shared = True
+        self.requires("openssl/[>=1.1.1n <2.0.0]")
```

Comment:
Conan upstream requirement can be overridden downstream. I intentional did so for openssl when integrating Celix with legacy application using Conan. May I have a look at the complete console output? I only have Ubuntu PC at hand.

IIRC, openssl does not use semantic versioning, thus version range does not work:

> There are some packages that do not follow semver. A popular one would be the OpenSSL package with versions as 1.0.2n. They cannot be used with version-ranges. 

https://docs.conan.io/en/latest/versioning/version_ranges.html?highlight=version%20range

Reply:
We can fixed openssl version to 1.1.1n due to the endless loop vulnerability in OpenSSL.